content/articles/visualizing-repositories-using-pygal/index.md

```diff
@@ -0,0 +1,371 @@
+Pygal allows us to create a range of graphs and charts. You'll learn how to construct a visualisation that shows the comparative popularity of Python works on GitHub in this tutorial. We'll create an interactive bar chart, where the height of each bar will represent the sum of stars earned by the project. By clicking a bar, you'll be sent to the project's GitHub page.
+
+### Prerequisites
+
+You'll need some Python skills to follow along with this tutorial. You must be in a position to work with web APIs in Python.
```

Comment:
```suggestion
You'll need some Python skills to follow along with this tutorial. You must be in a position to work with web APIs in Python.
```
Check on the prerequisites, a bit unclear. You could probably list the prerequisites such as PIP.